blog/2021/01/arrow.md

```diff
@@ -0,0 +1,32 @@
+@def title = "Apache Arrow Support in Julia"
+@def authors = "Jacob Quinn"
+@def published = "26 January 2021"
+@def rss_pubdate = Date(2021, 1, 26)
+@def rss = """Apache Arrow Support in Julia"""
+
+We're excited to announce official support for the Julia language in the [Apache Arrow]() project. Since the [1.0 release](https://arrow.apache.org/blog/2020/07/24/1.0.0-release/) of the arrow format last July (2020), there have been calls for support in Julia. While Julia supported the original [feather v1](https://github.com/JuliaData/Feather.jl) format very early on, support for the "upgraded" arrow format was delayed as the format matured.
```

Comment:
`[Apache Arrow]()` did you forget to insert a link to (I suppose) https://arrow.apache.org/ ?

Reply:
Indeed; good catch.